pkgs/applications/misc/snowsql/darwin.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  unzip,
+  cpio,
+  gzip,
+  unar,
+  darwin,
+  zlib,
+  version,
+  meta,
+  pname,
+}: let
+  arch =
+    if stdenv.isAarch64
+    then "arm64"
+    else "x86_64";
+in
```

The local variable `arch` can be replaced with `stdenv.hostPlatform.darwinArch` or `stdenv.targetPlatform.darwinArch`:
```suggestion
}:
```

I'd recommend removing the indentation or even better format the files using [nixfmt](https://github.com/NixOS/nixfmt): `nix run github:NixOS/nixfmt pkgs/applications/misc/snowsql/darwin.nix`